src/components/organisms/LinksBox.jsx

```diff
@@ -24,6 +24,33 @@ const LinksList = styled.div`
   height: auto;
   pointer-events: all;
 
+  @media ${MediaSmall} {
+    &.bottom-left {
+      justify-content: center;
+    }
+    &.bottom-right {
+      justify-content: center;
+    }
```

Comment:
hm, couldn't this be done in one line? 
e.g. 
```
&.bottom-left, &.bottom-right{
    justify-content: center;
}
```

Reply:
could indeed. Refactor candidate. :)